### PR TITLE
Reformat the file and remove semicolons

### PR DESCRIPTION
**Pull request by asankov**
to be consistent with the other parts of the file and of the new JS ones

### CODE REVIEW

File: tcms/testruns/static/testruns/js/get.js [hunk 1]
```diff
@@ -84,12 +84,12 @@ $(document).ready(() => {
...
 
     // bind everything in tags table
-    tagsCard('TestRun', testRunId, { run: testRunId }, permissions.removeTag);
+    tagsCard('TestRun', testRunId, { run: testRunId }, permissions.removeTag)
 
     jsonRPC('TestExecutionStatus.filter', {}, executionStatuses => {
         // convert from list to a dict for easier indexing later
-        for (var i = 0; i < executionStatuses.length; i++) {
-            allExecutionStatuses[executionStatuses[i].id] = executionStatuses[i];
+        for (var i = 0 i < executionStatuses.length i++) {
```
Comment (atodorov) [inline tcms/testruns/static/testruns/js/get.js:91]: SyntaxError: missing ; after for-loop initializer

**Review by atodorov — CHANGES_REQUESTED** [review]

File: tcms/testruns/static/testruns/js/get.js (hunk 1 above)
Comment (asankov) [inline tcms/testruns/static/testruns/js/get.js:91]:
```suggestion
        for (var i = 0; i < executionStatuses.length; i++) {
```

File: tcms/testruns/static/testruns/js/get.js [hunk 2]
```diff
@@ -495,7 +495,7 @@ function fileBugFromExecution(execution) {
             reloadRowFor(execution)
 
             // unescape b/c Issue #1533
-            const targetUrl = result.response.replace(/&amp;/g, '&')
+            const targetUrl = result.response.replace(/&amp/g, '&')
```
Comment (atodorov) [inline tcms/testruns/static/testruns/js/get.js:498]: bring back the `;`

**Review by atodorov — CHANGES_REQUESTED** [review]

File: tcms/testruns/static/testruns/js/get.js (hunk 2 above)
Comment (asankov) [inline tcms/testruns/static/testruns/js/get.js:498]:
```suggestion
            const targetUrl = result.response.replace(/&amp;/g, '&')
```

**Pull request merged**